README.rst

```diff
@@ -37,18 +39,33 @@ This code runs on Python 3.5 or newer.
 Docker
 ------
 
-Alternatively, you can build and run the xblock-sdk in Docker.
+Alternatively, you can build and run the xblock-sdk in Docker (we are using docker-compose which
+can be installed as explained at https://docs.docker.com/compose/install/)
 
 After cloning this repository locally, go into the repository directory and build the Docker image::
 
-    $ docker build -t xblock-sdk .
+    $ docker-compose build
 
 You can then run the locally-built version using the following command::
 
-    $ docker run -d -p 8000:8000 --name xblock-sdk xblock-sdk
+    $ docker-compose up -d
+
+and shut-down the container by::
+
+    $ docker-compose down
+
+On the first startup run the following command to create the SQLite database.
+(Otherwise you will get an error no such table: workbench_xblockstate.)
+Command::
+
+    $ docker container exec -it xblock-sdk_xblock_sdk_1 python3.8 manage.py migrate
```

Comment:
Can you put `#!/usr/bin/env python3` at the top of `manage.py`? That way we don't have to keep prepending `python` when running any commands in the shell.

Currently it leads to:
```
root@localhost:/usr/local/src/xblock-sdk# ./manage.py
./manage.py: line 1: Manage.py file for XBlock: command not found
./manage.py: line 2: import: command not found
./manage.py: line 3: import: command not found
./manage.py: line 5: import: command not found
./manage.py: line 8: syntax error near unexpected token `"DJANGO_SETTINGS_MODULE",'
./manage.py: line 8: `    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "workbench.settings")'
```

Reply:
Done.